doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -229,6 +229,27 @@ Securing Solr
 
 Our sample init script and systemd service file linked above tell Solr to only listen on localhost (127.0.0.1). We strongly recommend that you also use a firewall to block access to the Solr port (8983) from outside networks, for added redundancy.
 
+
+To start Solr Apache in external use mode, you must access the Solr example directory:
+
+# cd /usr/local/solr-7.7.2/bin/
+
+# ./solr start -j "-Djetty.host = DNS” -force
+
+Right after that you should use the iptables blocking rule
+
+# iptables -A INPUT -s ip address / 32 -p tcp -m tcp --dport 8983 -j ACCEPT
```

Comment:
This "ip address" is for the Dataverse server, right? To make this more clear, perhaps it could be IP_OF_DATAVERSE_SERVER.

Reply:
yes, IP_OF_DATAVERSE_SERVER i  fully agree.